test/unit/org/apache/cassandra/locator/EC2SnitchTest.java

```diff
@@ -135,22 +135,6 @@ public void testFullNamingScheme() throws IOException, ConfigurationException
         assertEquals("us-west-1a", snitch.getRack(local));
     }
 
-    @Test
-    public void validateDatacenter_RequiresLegacy_CorrectAmazonName()
-    {
-        Set<String> datacenters = new HashSet<>();
-        datacenters.add("us-east-1");
-        Assert.assertFalse(Ec2Snitch.validate(datacenters, Collections.emptySet(), true));
-    }
-
-    @Test
-    public void validateDatacenter_RequiresLegacy_LegacyName()
-    {
-        Set<String> datacenters = new HashSet<>();
-        datacenters.add("us-east");
-        Assert.assertTrue(Ec2Snitch.validate(datacenters, Collections.emptySet(), true));
-    }
-
```

Comment:
can we keep these tests, but change the verification? 
(it's nice to see what it does, without having to run the code for real)